pyproject.toml

```diff
@@ -1,61 +1,65 @@
 [build-system]
 build-backend = "hatchling.build"
-requires = ["hatchling>=1.12.2", "hatch-vcs>=0.3"]
+requires = [
+  "hatch-vcs>=0.3",
+  "hatchling>=1.18",
+]
 
 [project]
 name = "pytest-memray"
 description = "A simple plugin to use with pytest"
-readme.file = "README.md"
 readme.content-type = "text/markdown"
+readme.file = "README.md"
 license = "apache-2.0"
-urls."Bug Tracker" = "https://github.com/bloomberg/pytest-memray/issues"
-urls.Documentation = "https://pytest-memray.readthedocs.io"
-urls."Source Code" = "https://github.com/bloomberg/pytest-memray"
-authors = [
+maintainers = [
   { name = "Pablo Galindo Salgado", email = "pgalindo3@bloomberg.net" },
 ]
-maintainers = [
+authors = [
   { name = "Pablo Galindo Salgado", email = "pgalindo3@bloomberg.net" },
 ]
 requires-python = ">=3.8"
-dependencies = [
-  "pytest>=7.2",
-  "memray>=1.5",
-]
-optional-dependencies.docs = [
-  "furo>=2022.12.7",
-  "sphinx>=6.1.3",
-  "sphinx-argparse>=0.4",
-  "sphinx-inline-tabs>=2022.1.2b11",
-  "sphinxcontrib-programoutput>=0.17",
-  "towncrier>=22.12",
-]
-optional-dependencies.lint = [
-  "black==22.12",
-  "flake8==6",
-  "isort==5.11.4",
-  "mypy==0.991",
-]
-optional-dependencies.test = [
-  "covdefaults>=2.2.2",
-  "pytest>=7.2",
-  "coverage>=7.0.5",
-  "flaky>=3.7",
-  "pytest-xdist>=3.1",
-]
-dynamic = ["version"]
 classifiers = [
   "Intended Audience :: Developers",
   "License :: OSI Approved :: Apache Software License",
   "Operating System :: POSIX :: Linux",
+  "Programming Language :: Python :: 3 :: Only",
   "Programming Language :: Python :: 3.8",
   "Programming Language :: Python :: 3.9",
   "Programming Language :: Python :: 3.10",
   "Programming Language :: Python :: 3.11",
   "Programming Language :: Python :: Implementation :: CPython",
   "Topic :: Software Development :: Debuggers",
 ]
-
+dynamic = [
+  "version",
+]
+dependencies = [
+  "memray>=1.8",
```

Comment:
Why are we changing from depending on memray >=1.5 to >=1.8? Is this something that should have already been applied in the past and was missed? If so, can you at least make it a separate commit so we can find it when searching through the git log?

Reply:
Because the CI is only testing the latest version of memray. Anything else is a guess at best, and a straight lie otherwise.

Reply:
OK, fine. This should be done in a separate commit, so that your reviewer doesn't need to guess what changes were made automatically by a tool and what changes were made manually by you.